docs/arch/readme.md

```diff
@@ -0,0 +1,81 @@
+# Architecture Decision Records
+
+_ADR_ for short.
+
+## What is an Architecturally Significant Decision?
+
+See [Documenting Architecture
+Decisions](http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions)
+for more information.
+
+The basic idea is to capture key decisions having to do with anything _architectural_
+in a way that promotes better communication than simple word-of-mouth.
+
+What is an _architectural_ decision?  If one or more of the following ideas apply you
+might be dealing with an architectural decision.
+
+Does the design decision...
+
+* Alter externally visible system properties?
+* Modify public interfaces?
+* Directly influence high priority quality attributes?
+* Include or remove dependencies?
+* Result from a discussion where you learned more about technical or business constraints?
+* Involve taking on strategic technical debt?
+* Change the structures of the system (static, dynamic, or physical)?
+* Require other developers to update construction techniques or development environments?
+
+
+## Template
+
+Use this template in any new ADRs.  Replace the help text as you write the ADR.
+
+```
+# ADR N: Brief Decision Title
+
+Context goes here.
+
+Describe the forces at play, including technological, political, social, and project local.
+These forces are likely in tension, and should be called out as such. The language in this
+section is value-neutral. It is simply describing facts.
+
+## Decision
+
+This section describes our response to these forces. It is stated in full sentences,
+with active voice. "We will ..."
+
+
+## Status
+
+choose one: [Proposed | Accepted | Deprecated | Superseded]
+
+if deprecated, include a rationale.
+
+If superseded, include a link to the new ADR
+
+
+## Consequences
+
+Describe the resulting context, after applying the decision. All consequences should be listed here,
+not just the "positive" ones. A particular decision may have positive, negative, and neutral consequences,
+but all of them affect the team and project in the future.
+```
```

Comment:
I would add [this ADR tools](https://adr.github.io/#decision-capturing-tools) link for people who would prefer to generate the ADR using the CLI.